Containerfile-dev

```diff
@@ -0,0 +1,9 @@
+FROM quay.io/389ds/ci-images:fedora
+ARG CUSTOM_CRT_URL
+
+RUN if [ -z "$CUSTOM_CRT_URL" ] ; then echo "No custom cert needed"; else \
```

Comment:
The build can fail when run behind a corporate intercepting TLS proxy because the build fetches files from the Internet.  This is a common issue for me at least.  I often create Dockerfiles at home on my Windows PC (or travel Macbook) that are fine, but then don't work as soon I try to run them onsite at work on RHEL.  Their may be a cleaner way to handle this, but for the most flexibility allowing  a user to pass in via environment variable the path to their site's local PKI cert is nice.   So for me, the build fails at work unless I have a .env file containing: 

```
CUSTOM_CRT_URL=https://pki.jlab.org/JLabCA.crt
```

I suppose we could just add a bash script that users could execute at run-time to set trusted certs (or otherwise just provide tips in the README on how to update the trust anchors).  Trying to make the image easy to use for all.  Let me know how you'd prefer to proceed.

Reply:
To clarify, the dirsrv build definately will fail behind our corporate firewall.  I think the Docker build is fine though at least at our site, not sure if someone else's site proxy intercepts dnf file fetches in a breaking way.  I have had other docker builds fail behind the proxy due to yum/dnf/apt get, whatever.